research-software-directory/RSD-as-a-service
# Migrate topics for projects

# Migrate topics for projects

Changes proposed in this pull request:

* migrate tags for projects
* migrate topics for projects
* improve build process for data-migration and authentication: compilation now happens during build instead of when starting

How to test:

* After building and running, the pages http://localhost:3500/tag_for_project and http://localhost:3500/topic_for_project should be populated. You can also do join queries like http://localhost:3500/project?select=id,slug,tag_for_project(tag),topic_for_project(topic)
* You should see successful compilation messages when building data-migration and authentication

PR Checklist:
